checkov/kubernetes/graph_builder/graph_components/K8SEdgeBuilder.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+from abc import abstractmethod
+
+from checkov.kubernetes.graph_builder.graph_components.blocks import KubernetesBlock
+
```

Comment:
please add here an extra new line, otherwise I will need to adjust it in my flake8 PR 😄